e2e/framework/testserver.go

```diff
@@ -490,12 +509,16 @@ func (t *TestServer) WaitForReady(ctx context.Context) error {
 	return err
 }
 
-func (t *TestServer) TxnTo(ctx context.Context, address web3.Address, method string) *web3.Receipt {
+//nolint:lll
```

Comment:
Remove the `nolint`, just break up the method declaration on multiple lines so it's readable

Reply:
Fixed in 6138b95b4526bbbefa2f06a54581e344e2ebc9cd